source/features/index.tsx

```diff
@@ -105,9 +105,11 @@ const globalReady: Promise<RGHOptions> = new Promise(async resolve => {
 
 	// Options defaults
 	const options = await optionsStorage.getAll();
-
-	// If features are remotely marked as "seriously breaking" by the maintainers, disable them without having to wait for proper updates to propagate #3529
-	const hotfix = await cache.get('hotfix');
+	const hotfix = browser.runtime.getManifest().version === '0.0.0' || await cache.get('hotfix'); // Ignores the cache when loaded locally
+	// if features are marked as "seriously breaking" by maintainers
+	// disable them without having to wait for fixed version updates
+	// to propogate to browsers
+	// refer GitHub issue #3529 for more information
```

Comment:
Why did you badly rewrite this comment? I had already written it.

Reply:
I'll revert my changes.

Reply:
done. and all checks are passing. thanks for updating the branch with the default branch. I should've thought of that before calling the test failure a 'false positive'.